C/Clipper2/bundled/patches/sharedlib.patch

```diff
@@ -1,61 +1,52 @@
 diff --git before/Clipper2/CPP/CMakeLists.txt after/Clipper2/CPP/CMakeLists.txt
-index bb1e924..1b1860b 100644
+index e3afbc8..87ffa25 100644
 --- before/Clipper2/CPP/CMakeLists.txt
 +++ after/Clipper2/CPP/CMakeLists.txt
-@@ -25,7 +25,7 @@ set(CLIPPER2_SRC
- )
- 
- # 2d version of Clipper2
--add_library(Clipper2 STATIC ${CLIPPER2_INC} ${CLIPPER2_SRC})
-+add_library(Clipper2 SHARED ${CLIPPER2_INC} ${CLIPPER2_SRC})
```

Comment:
I think this is the library we're missing?

Reply:
You mean this should be put inside the currently-empty `dependencies = Dependency[]` of the `build_tarballs.jl` file?

Reply:
He means that this patch for this new version should still include these changes to the CMake file to generate the `Clipper2` library.